### PR TITLE
fix(Nodes): use `$adm-colors` instead of Vuetify colors

### PR DESCRIPTION
**Pull request by joebrock666**
https://trello.com/c/AA1z69S5

### CODE REVIEW

File: src/components/NodesTable/NodesTableItem.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@
         {{ nodeStatus }}
       </span>
       <v-icon
-        :class="classes.statusIcon"
+        :class="`${classes.statusIcon} ${nodeStatusColor}`"
         :color="nodeStatusColor"
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:31]: remove

File: src/components/NodesTable/NodesTableItem.vue [hunk 2]
```diff
@@ -27,7 +27,7 @@
         {{ nodeStatus }}
       </span>
       <v-icon
-        :class="classes.statusIcon"
+        :class="`${classes.statusIcon} ${nodeStatusColor}`"
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:30]: use array

File: src/components/NodesTable/NodesTableItem.vue [hunk 3]
```diff
@@ -166,6 +166,19 @@ export default {
   }
 }
 
+.green {
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:169]: &__status-icon {
  &--green {
      color: ...
  }
  ...
}

File: src/components/NodesTable/NodesTableItem.vue [hunk 4]
```diff
@@ -166,6 +166,19 @@ export default {
...
+  color: rgb(61, 209, 151) !important;
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:170]: use $adm-colors

File: src/components/NodesTable/NodesTableItem.vue [hunk 5]
```diff
@@ -166,6 +166,19 @@ export default {
...
+}
+.red {
+  color: rgb(237, 82, 112) !important;
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:173]: use $adm-colors

File: src/components/NodesTable/NodesTableItem.vue [hunk 6]
```diff
@@ -166,6 +166,19 @@ export default {
...
+}
+.grey {
+  color: #9e9e9e !important;
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:176]: use $adm-colors

File: src/components/NodesTable/NodesTableItem.vue [hunk 7]
```diff
@@ -166,6 +166,19 @@ export default {
...
+}
+.orange {
+  color: rgb(248, 160, 97) !important;
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:179]: use $adm-colors

File: src/components/NodesTable/NodesTableItem.vue (hunk 3 above)
Comment (bludnic) [reply src/components/NodesTable/NodesTableItem.vue:169]: create same selector in .v-theme--light and .v-theme--dark

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/components/NodesTable/NodesTableItem.vue [hunk 8]
```diff
@@ -143,6 +142,18 @@ export default {
...
+    }
+    &--red {
+      color: rgb(237, 82, 112) !important;
+    }
+    &--grey {
+      color: #9e9e9e !important;
+    }
+    &--orange {
+      color: rgb(248, 160, 97) !important;
+    }
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue]: Remove this. You must specify colors only in `.v-theme--light` and `.v-theme--dark` selectors.

File: src/components/NodesTable/NodesTableItem.vue [hunk 9]
```diff
@@ -37,7 +36,7 @@
     <td :class="classes.td" class="pl-0 pr-2">
       <v-icon
         :icon="socketSupport ? 'mdi-check' : 'mdi-close'"
-        :class="socketSupport ? 'green' : 'red'"
+        :class="socketSupport ? `${classes.statusIcon}--green` : `${classes.statusIcon}--red`"
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:39]: Define class modifiers below in the `const classes`.

```js
    const classes = {
      // ...
      statusIcon: `${className}__status-icon`,
      statusIconGreen: `${className}__status-icon--green`,
      statusIconRed: `${className}__status-icon--red`,
      // ...
    }
```

File: src/components/NodesTable/NodesTableItem.vue [hunk 10]
```diff
@@ -9,8 +9,7 @@
     >
       <v-checkbox-btn
         :model-value="active"
-        :class="classes.checkbox"
-        class="grey"
+        :class="[classes.checkbox, `${classes.statusIcon}--grey`]"
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:12]: :class="[classes.checkbox, classes.statusIconGrey]"

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/components/NodesTable/NodesTableItem.vue [hunk 11]
```diff
@@ -37,7 +42,7 @@
     <td :class="classes.td" class="pl-0 pr-2">
       <v-icon
         :icon="socketSupport ? 'mdi-check' : 'mdi-close'"
-        :color="socketSupport ? 'green' : 'red'"
+        :class="socketSupport ? `${classes.statusIcon}--green` : `${classes.statusIcon}--red`"
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:45]: classes.statusIconGreen and classes.statusIconRed

File: src/components/NodesTable/NodesTableItem.vue [hunk 12]
```diff
@@ -75,7 +80,7 @@ function getNodeStatusColor(node) {
     color = 'orange'
   }
 
-  return color + ' lighten-1'
+  return color;
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:83]:
```suggestion
  return color
```

File: src/components/NodesTable/NodesTableItem.vue [hunk 13]
```diff
@@ -27,7 +26,13 @@
...
       </span>
       <v-icon
-        :class="classes.statusIcon"
+        :class="{
+        [classes.statusIcon]: true,
+        [classes.statusIconGreen]: nodeStatusColor === 'green',
+        [classes.statusIconRed]: nodeStatusColor === 'red',
+        [classes.statusIconOrange]: nodeStatusColor === 'orange',
+        [classes.statusIconGrey]: nodeStatusColor === 'grey'
+    }"
```
Comment (bludnic) [inline src/components/NodesTable/NodesTableItem.vue:35]: prettify

**Review by bludnic — CHANGES_REQUESTED** [review]

**Comment by bludnic** [comment]
@joebrock666 Rollback package-lock.json. Use node v18

File: src/components/NodesTable/NodesTableItem.vue (hunk 11 above)
Comment (bludnic) [reply src/components/NodesTable/NodesTableItem.vue:45]: not fixed

**Comment by bludnic** [comment]
There are still changes in the `package-lock.json`. Fix it @joebrock666

**Comment by joebrock666** [comment]
> There are still changes in the `package-lock.json`. Fix it @joebrock666

I think now it's the same as in dev

**Review by bludnic — APPROVED** [review]

**Pull request merged**